### DISCUSSION TITLE
No Benefits button on Github Education page

### DISCUSSION THREAD

**Discussion by Kobe972** [community/community#130691] [GitHub Education]
### Hi everyone,

https://github.com/edu has not Benefits button on the top navigator

**Comment by davevad93** [comment]
Hi @Kobe972 ,

If you are looking for the benefits application page, here it is:

- [Access free GitHub Education benefits](https://education.github.com/discount_requests/application)

**Reply by Kobe972** [reply]
I opened the link and it redirected to https://github.com/pricing

**Reply by davevad93** [reply]
Hey @Kobe972 , that's weird the link I gave you is the benefits application page and should redirect you to "`https://education.github.com/discount_requests/application`" not on the pricing page.

**Reply by Kobe972** [reply]
Yes, that's wierd. It only redirects for this account.
